### DISCUSSION TITLE
DNS lookup failed

### DISCUSSION THREAD

**Discussion by 61743** [nxtrace/NTrace-core#230] [General]
root@885-2023052012473607:~# nexttrace -6 2600:1900:41a0:cb2a::1 -T -p 22
NextTrace v1.2.8 2023-12-23T13:30:40Z f76c940
[NextTrace API] preferred API IP - 104.21.40.176 - 116.32ms - Misaka.HKG
DNS lookup failed

**Comment by tsosunchia** [comment]
😅你自己输错了 再仔细看看
